src/mapgen/treegen.cpp

```diff
@@ -876,4 +876,16 @@ void make_pine_tree(MMVManip &vmanip, v3s16 p0, const NodeDefManager *ndef,
 	}
 }
 
+const std::string error_to_string(error e) {
```

Comment:
also returning const is pointless with bare values
```suggestion
const std::string error_to_string(error e)
{
```